src/app.js

```diff
@@ -1,21 +1,36 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const fs = require('fs');
+
+const renameFile = (oldName, newName, isExist) => {
```

Comment:
you can just handle it before you call the function
```suggestion
const renameFile = (oldName, newName) => {
```